### ISSUE TITLE
GraphqlGoType definition is inconsistent for google protobuf wrapper types

### ISSUE THREAD

**Issue by saichander** [ysugimoto/grpc-graphql-gateway#28]
All the wrapper types are defined in the wrappers package and are imported in the generated .graphql.go file with package name alias `gql_ptypes_wrappers`. However, when referring to the graphql objects in the generated .graphql.go file, it is mapped incorrectly

- For Query Types, the field type is correctly mapped using `gql_ptypes_wrappers.Gql__type_Int32Value()`

- For Input Types and in Mutations, these are incorrectly mapped to `gql_ptypes_int32value.Gql__input_Int32Value()`, `gql_ptypes_int32value` package is invalid.

I did quick debugging and found that the `GraphqlGoType()` function in the `spec` package is the culprit. It is deducing different types based on the isInput flag. For input types, it is prefixing the package name as `gql_ptypes_<actual field type>`(eg: gql_ptypes_int32value) and for query types, it is setting package prefix as `gql_ptypes_wrappers`.

```
case descriptor.FieldDescriptorProto_TYPE_MESSAGE:
		m := f.DependType.(*Message) // nolint: errcheck
		tn := strings.TrimPrefix(f.TypeName(), m.Package()+".")
		if f.IsCyclic {
			return PrefixInterface(strings.ReplaceAll(tn, ".", "_"))
		}
		**if isInput {
			if IsGooglePackage(m) {
				return PrefixPtypesInput(strings.ReplaceAll(tn, ".", "_"))
			}
			return PrefixInput(strings.ReplaceAll(tn, ".", "_"))
		}**
		var pkgPrefix string
		pkg := NewPackage(m)
		if IsGooglePackage(m) {
			name := strings.ToLower(filepath.Base(m.GoPackage()))
			mustImplementedPtypes(name)
			**pkgPrefix = "gql_ptypes_" + name + "."**
		} else if rootPackage != "." {
			// Case message is nested, also includes map_entry
			if pkg.Name != rootPackage {
				if IsGooglePackage(m) {
				} else {
					pkgPrefix = pkg.Name + "."
				}
			}
		}
		return pkgPrefix + PrefixType(strings.ReplaceAll(tn, ".", "_"))
```
As I am not sure why the behavior is different, I am reaching out to you for help. Thanks

**Comment by saichander** [comment]
@ysugimoto Can you look into it when you get some time?

**Comment by ysugimoto** [comment]
@saichander Hi again, thank you for finding additional problem.


I fixed that problem quickly on https://github.com/ysugimoto/grpc-graphql-gateway/pull/29 and please find the plugin binary in that PR's comment and I'm happy if you could try to generate code.

**Comment by tarunvelli** [comment]
@ysugimoto @saichander, I tested the above and saw that there is another unrelated issue that should be fixed by https://github.com/ysugimoto/grpc-graphql-gateway/pull/30

**Comment by ysugimoto** [comment]
@tarunvelli Thanks, If your PR is not related to this PR, let's discuss on #30 👍

**Comment by ysugimoto** [comment]
Just merged and published as `v0.18.2`.
